tests/HasAttributesTest.php

```
@@ -29,91 +30,182 @@
+            'Should handle Stringable objects correctly.',
+        );
+    }
+
+    public function testAttributesPrefixSupportThroughProtectedInternals(): void
+    {
+        $instance = new class {
+            use HasAttributes;
+
+            public function getAttributeForTest(string|UnitEnum $key, mixed $default = null, string $prefix = ''): mixed
+            {
+                return \UIAwesome\Html\Helper\AttributeBag::get($this->attributes, $key, $default, $prefix);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'testAttributesPrefixSupportThroughProtectedInternals'. (undefined)

(StaticAccess)